Pod/Core/TypeConversion.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  TypeConversion.swift
+//  Pods
+//
+//  Created by Sam Dean on 4/4/17.
+//
+//
+
+import Foundation
+
+
+public protocol FromStringable {
```

Comment:
The bit that was news to me was that you can do this: `extension MatchedStringRepresentable where Self: LosslessStringConvertible` - extending a protocol based on the type of the class implementing it. Nice work Swift :)